src/src/Commands/InternalCommands_decoder.cpp

```diff
@@ -276,6 +276,10 @@ const char Internal_commands_w[] PROGMEM =
   "wdconfig|"
   "wdread|"
 #endif // ifndef LIMIT_BUILD_SIZE
+#ifdef FEUTER_CAN

```

Comment:
- That's probably a typo, should better be `#if FEATURE_CAN`
- We've stopped using `#ifdef` for feature-flags, to make handling defaults possible, that can also be controller from a `Custom.h` file, to be explicitly enabled (1) or disabled (0)